AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/test/Storage/TestGetItemsForInitializeMutation.dfy

```diff
@@ -39,29 +39,60 @@ module {:options "/functionSyntax:4"} TestGetItemsForInitializeMutation {
       output.BeaconItem.Type.ActiveHierarchicalSymmetricBeacon?,
       "beaconItem was not Beacon? 'type': " + output.BeaconItem.EncryptionContext[Structure.TYPE_FIELD];
     expect
-      output.MutationLock.None?,
-      "MutationLock was not None. 'UUID': " + output.MutationLock.value.UUID;
+      output.MutationCommitment.None?,
+      "MutationCommitment was not None. 'UUID': " + output.MutationCommitment.value.UUID;
+    expect
+      output.MutationIndex.None?,
+      "MutationIndex was not None. 'UUID': " + output.MutationIndex.value.UUID;
   }
 
   method {:test} TestHappyCaseMLocked()
```

Comment:
remove?

Reply:
No. I want to restore this test.
I just want to do it later,
after I have implemented more of the source code.